addon/components/nrg-home-cards/home-card.hbs

```diff
@@ -1,17 +1,15 @@
 <a
-  role="button"
   class="card home-card
     {{if @actionCard "action-card"}}
     {{unless this.visible "is-visually-hidden"}}"
-  {{on "click" this.click}}
```

Comment:
Why did you delete this? @tsmalls93 added it in #330

Reply:
This was certainly a mistake caused when I rebased.